microsoft-365/security/office-365-security/install-app-guard.md

```diff
@@ -114,7 +118,9 @@ Upon being opened, the file should display a few visual indicators that the file
 
 ## Configure Application Guard for Office
 
-Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the Office cloud policy service.
+Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office Cloud Policy Service] (/DeployOffice/overview-office-cloud-policy-service).
```

Comment:
double-check in preview if the space between ] and ( affects render of hyperlink